hack/update-codegen.sh

```diff
@@ -26,3 +26,15 @@ ${SEDNA_ROOT}/hack/generate-groups.sh "deepcopy,client,informer,lister" \
 ${SEDNA_GO_PACKAGE}/pkg/client ${SEDNA_GO_PACKAGE}/pkg/apis \
 "sedna:v1alpha1" \
 --go-header-file ${SEDNA_ROOT}/hack/boilerplate/boilerplate.generatego.txt
+
+# Check if Sedna home is different from the standard directory where GO projects are located
+if [ "${GOPATH}/src/${SEDNA_GO_PACKAGE}/" != "$${SEDNA_ROOT}/" ]; then
```

Comment:
1.  $${SEDNA_ROOT} =>  ${SEDNA_ROOT}
2.  we need to check they are the same location instead of only checking equal string, such as symbol link relation

Reply:
More word, the user may create the symbol link directory for $SEDNA_ROOT in ${GOPATH}/src